src/TimeSlot.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes, Component } from 'react'
+import cn from 'classnames'
+
+export default class TimeSlice extends Component {
+  static propTypes = {
+    value: PropTypes.instanceOf(Date).isRequired,
+    isNow: PropTypes.bool,
+    showLabel: PropTypes.bool,
+    selected: PropTypes.bool,
+    content: PropTypes.string,
+    culture: PropTypes.string
+  }
+
+  static defaultProps = {
+    isNow: false,
+    showLabel: true,
```

Comment:
can we make `showLabel` have no default (or default `false`) since its the more common value.